### PR TITLE
If Bazel is installed on a builder, build test_bazel as well

### PR DESCRIPTION
**Pull request by steven-johnson**

### CODE REVIEW

**Comment by steven-johnson** [comment]
Monday morning review ping

File: master/master.cfg [hunk 1]
```diff
@@ -336,6 +337,10 @@ def get_targets(os, llvm):
                         ('test_generators', t),
                         ('test_apps', t)])
 
+    # If Bazel is installed on the builder, run test_bazel
```
Comment (abadams) [inline master/master.cfg]: I think this code runs on the master, not the builder. You probably just want to just condition it on the os starting with 'linux-64'

Comment (steven-johnson) [inline master/master.cfg]: Done

**Comment by abadams** [comment]
I just did a git pull on the build master and restarted it. Testing should commence now. Sorry for the delay - busy at siggraph.

**Pull request merged**